src/SFA.DAS.AdminService.Web.Tests/Services/Gateway/OrganisationChecks/Orchestrator/LegalNameTests.cs

```diff
@@ -13,7 +14,7 @@
 namespace SFA.DAS.AdminService.Web.Tests.Services.Gateway.OrganisationChecks.Orchestrator
 {
     [TestFixture]
-    public class LegalNameTests
+    public class SubcontractorDeclarationTests
```

Comment:
This name change looks wrong

Reply:
I'm blaming ReSharper! 😁 Reverted that change.